backend/lcfs/web/api/compliance_report/repo.py

```diff
@@ -67,7 +67,16 @@ def apply_filters(self, pagination, conditions):
             filter_type = filter.filter_type
             if filter.field == "status":
                 field = get_field_for_filter(ComplianceReportStatus, "status")
-                filter_value = getattr(ComplianceReportStatusEnum, filter_value.title())
+                # Normalize the status value by making it lowercase and replacing spaces with underscores
+                normalized_value = filter_value.replace(" ", "_").lower()
+
+                # Match enum values without case-sensitivity
+                matching_enum_value = None
+                for enum_val in ComplianceReportStatusEnum:
+                    if enum_val.name.lower() == normalized_value:
```

Comment:
code smell: It's odd to use enum_val.name, remember that this gets the name of the enum variable not the value of the enum. You should probably be using the enum itself most of the time, as changing the name of an enum should not break code.

Reply:
Thanks, Daniel, for pointing this out, I fixed the issue.